### ISSUE TITLE
Process some small experiments explicitly for "kicking the identifier conversion tires"

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/refinebio#466]
### Context

Once https://github.com/AlexsLemonade/refinebio/pull/447 lands, we should be able to do the following identifier conversions for submitter-processed data:

* Illumina BeadArray: Probe ID to Ensembl gene ID (note: not in that PR)
* Affymetrix: Probe ID,  Entrez ID, RefSeq, Unigene, or gene symbol to Ensembl gene ID (see: https://github.com/AlexsLemonade/identifier-refinery)

### Problem or idea

@Miserlou suggested we do some analyses with data that's been processed with this pipeline before we close #186, #328, #329, and #348. 

### Solution or next step

I've picked out a few experiments for this purpose.

* Brainarray-supported Affymetrix, has raw data [`GSE12021`](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE12021)
* Brainarray-supported Affymetrix, no raw data: [`GSE56409`](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56409)
* Illumina, has raw data: [`GSE44719`](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE44719)
* Illumina, no raw data: [`E-GEOD-44719`](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-44719/) (note that this is the same experiment from above, but from ArrayExpress -- the raw data might be an "additional data" file at AE; let me know if this won't work)
* Affymetrix PM Array Plate, has raw data: [`GSE77094`](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE77094)
* Affymetrix PM Array Plate, no raw data: [`E-GEOD-77094`](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-77094/) (similar to above, where the AE accession doesn't have raw data)

These should be run after #447 lands. They don't need to be run in production, I just need to get my hands on them.

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [x] You have reported a problem or idea.
- [x] You have proposed a solution or next step.

**Comment by Miserlou** [comment]
Excellent test list, thank you Jackie. We should probably be able to take care of this for you tomorrow.

The list for our file:
```
GSE12021
GSE56409
GSE44719
E-GEOD-44719
E-GEOD-77094
```

These are also good candidates for including in an e2e test.

**Comment by jaclyn-taroni** [comment]
If we intend to have me retrieve this from the UI, can we do so such that I can also test out the transformations with the transformation dropdown menu at the same time?

**Comment by kurtwheeler** [comment]
Is that option fulling working but we just hid it for the sake of the training session so as to not confused people?

cc @Willv19 @Miserlou

**Comment by jaclyn-taroni** [comment]
This lead me to believe that it’s ready: https://github.com/AlexsLemonade/refinebio-frontend/issues/208

But I could be mistaken

**Comment by Willv19** [comment]
The transformation dropdown is fully working on the frontend.

**Comment by jaclyn-taroni** [comment]
I don't think these are likely to get included in the zebrafish crunch. I will attempt to run these locally this week.

**Comment by Miserlou** [comment]
Processed manually, seems mostly okay. However, I got this on `GSE44719`:

```
2018-09-21 16:13:01,279 local data_refinery_workers.processors.utils DEBUG [processor_job: 5] [pipeline: ILLUMINA_TO_PCL]: Starting processor Job.
Parallel computing support for 'oligo/crlmm': Disabled
     - Load 'ff'
     - Load and register a 'foreach' adaptor
        Example - Using 'multicore' for 2 cores:
             library(doMC)
             registerDoMC(2)
================================================================================
Reading data file...
Processing 4424548043_L
Processing 4447846378_A
Processing 4447846334_D
Processing 4447846382_C
Error in unserialize(node$con) : error reading from connection
Calls: <Anonymous> ... doTryCatch -> recvData -> recvData.SOCKnode -> unserialize
Execution halted
2018-09-21 16:17:35,563 local data_refinery_workers.processors.illumina ERROR [processor_job: 5]: Encountered error in R code while running illumina.R pipeline during processing of /home/user/data_store/processor_job_5/GSE44719_non_normalized.txt.sanitized: Command '['/usr/bin/Rscript', '--vanilla', '/home/user/data_refinery_workers/processors/illumina.R', '--probeId', 'ID_REF', '--expression', '26,98,114,126,48,38,102,82,148,14,88,46,144,66,112,44,16,122,146,140,32,76,118,58,6,28,132,130,134,138,110,78,56,60,30,2,96,100,4,90,36,24,86,80,124,108,74,10,106,70,94,40,18,120,64,116,52,34,92,20,22,154,104,54,84,128,142,62,12,72,152,50,42,150,68,136,8', '--detection', 'Detection Pval', '--platform', 'illuminaHumanv3', '--inputFile', '/home/user/data_store/processor_job_5/GSE44719_non_normalized.txt.sanitized', '--outputFile', '/home/user/data_store/processor_job_5/GSE44719_non_normalized.PCL', '--cores', '4']' returned non-zero exit status -9
2018-09-21 16:17:35,630 local data_refinery_workers.processors.utils ERROR [processor_job: 5] [failure_reason: Encountered error in R code while running illumina.R pipeline during processing of /home/user/data_store/processor_job_5/GSE44719_non_normalized.txt.sanitized: Command '['/usr/bin/Rscript', '--vanilla', '/home/user/data_refinery_workers/processors/illumina.R', '--probeId', 'ID_REF', '--expression', '26,98,114,126,48,38,102,82,148,14,88,46,144,66,112,44,16,122,146,140,32,76,118,58,6,28,132,130,134,138,110,78,56,60,30,2,96,100,4,90,36,24,86,80,124,108,74,10,106,70,94,40,18,120,64,116,52,34,92,20,22,154,104,54,84,128,142,62,12,72,152,50,42,150,68,136,8', '--detection', 'Detection Pval', '--platform', 'illuminaHumanv3', '--inputFile', '/home/user/data_store/processor_job_5/GSE44719_non_normalized.txt.sanitized', '--outputFile', '/home/user/data_store/processor_job_5/GSE44719_non_normalized.PCL', '--cores', '4']' returned non-zero exit status -9]: Processor function _run_illumina failed. Terminating pipeline.
2018-09-21 16:17:35,944 local data_refinery_workers.processors.utils ERROR [no_retry: False] [processor_job: 5] [pipeline_applied: ILLUMINA_TO_PCL] [failure_reason: Encountered error in R code while running illumina.R pipeline during processing of /home/user/data_store/processor_job_5/GSE44719_non_normalized.txt.sanitized: Command '['/usr/bin/Rscript', '--vanilla', '/home/user/data_refinery_workers/processors/illumina.R', '--probeId', 'ID_REF', '--expression', '26,98,114,126,48,38,102,82,148,14,88,46,144,66,112,44,16,122,146,140,32,76,118,58,6,28,132,130,134,138,110,78,56,60,30,2,96,100,4,90,36,24,86,80,124,108,74,10,106,70,94,40,18,120,64,116,52,34,92,20,22,154,104,54,84,128,142,62,12,72,152,50,42,150,68,136,8', '--detection', 'Detection Pval', '--platform', 'illuminaHumanv3', '--inputFile', '/home/user/data_store/processor_job_5/GSE44719_non_normalized.txt.sanitized', '--outputFile', '/home/user/data_store/processor_job_5/GSE44719_non_normalized.PCL', '--cores', '4']' returned non-zero exit status -9]: Processor job failed!
```

Had to fix a bug for the SDRF harmonizer because that metadata is weird on that one, but it worked after that. Everything else worked just fine.

**Comment by Miserlou** [comment]
Internet suggests it could be a lack-of-RAM issue? https://stackoverflow.com/a/31789312/1135467

I ran this locally. I have 16GB total but who knows how much I had free. Probably none because of Slack.

**Comment by jaclyn-taroni** [comment]
Do you have the ArrayExpress noop files handy locally `E-GEOD-44719` and `E-GEOD-77094` @Miserlou
